src/api/c/transform.cpp

```diff
@@ -160,12 +160,12 @@ af_err af_translate(af_array *out, const af_array in, const float trans0, const
 {
 
     try {
-        static float trans_mat[6] = {1, 0, 0,
-                                     0, 1, 0};
+        thread_local float trans_mat[6] = {1, 0, 0,
```

Comment:
there is almost no point to make this thread local. We aren't gaining much by saving its state.